METDEV-1362 upstream/develop sync

flx-sta opened the pull request:
Für die Frontend-Tests den gleichen branch (METDEV-1362) im "metaproc" repo auschecken da diese sonst fehlschlagen

reusr1 committed:
Feature/askuser (#193)

* support for ask user dialog

Commit:
Fixed an issue with mapped Image Paths. Now we only compare the file name between Asset and Request instead of the entire path

Commit:
Merge branch 'develop' of https://github.com/headwirecom/peregrine-cms into METDEV-1362

Commit:
Added the Image’s Dimension to the Uploaded Asset’s Metadata (per-data)

reusr1 committed:
#155: fix for issue where initial state of component filter does not work

reusr1 committed:
#208: make sure insane large numbers are not causing the page to fail to render

andreas-hauser committed:
Insource external shim.min.js file (#204)

* METDEV-1658: Move shim.min.js to pagerenderer module
- make sure to keep it separate from the other JS files, to avoid it being included in the combined JS file.

DimitriPlotnikov commented:
@Felix-Puetz 
Frontend tests fail
```
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   DemocontentContainerAccordionIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[ERROR]   MetaprocdeutschContainerAccordionIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[ERROR]   DemocontentContainerSelectIT>ComponentBase.tearDown:54->ComponentBase.removeTestImage:200 » NoAlertPresent
[ERROR]   MetaprocdeutschContainerSelectIT>ComponentBase.tearDown:54->ComponentBase.removeTestImage:200 » NoAlertPresent
[ERROR]   DemocontentHeadlineSelectIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[ERROR]   MetaprocdeutschHeadlineSelectIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[ERROR] com.metaproc.frontendtests.publicwebsite.pageeditor.component.horizontalline.democontent.DemocontentHorizontalLineIT.testComponent(com.metaproc.frontendtests.publicwebsite.pageeditor.component.horizontalline.democontent.DemocontentHorizontalLineIT)
[ERROR]   Run 1: DemocontentHorizontalLineIT>ComponentBase.setUp:40->ComponentBase.testCreatePage:160 » Timeout
[ERROR]   Run 2: DemocontentHorizontalLineIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[INFO] 
[ERROR]   MetaprocdeutschHorizontalLineIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[ERROR]   DemocontentRichheadlineIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[ERROR]   MetaprocdeutschRichheadlineIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[ERROR]   DemocontentRichtextIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[ERROR]   MetaprocdeutschRichtextIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[ERROR]   DemocontentRowIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[ERROR]   MetaprocdeutschRowIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[ERROR]   DemocontentSpacerIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[ERROR]   MetaprocdeutschSpacerIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[ERROR]   DemocontentTextSelectIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[ERROR]   MetaprocdeutschTextSelectIT>ComponentBase.tearDown:57->ComponentBase.removePage:190 » NoAlertPresent
[INFO] 
[ERROR] Tests run: 19, Failures: 0, Errors: 18, Skipped: 0

```

flx-sta commented:
@DimitriPlotnikov hier muss parallel auch der gleiche Branch im metaproc repo ausgecheckt sein

tsaurabh74 committed:
Fixed typos. (#216)

bkheadwire committed:
Added backend functionality for updating a site from its theme to get any newly added components. (#187)

reusr1 committed:
#209: make sure components can only be dropped in the right place

dna113p committed:
Feature/resize observer polyfill (#222)

* convert height change listener to ResizeObsever
* add a resize observer polyfill
* move polyfill into document write

Commit:
#165 - Fixed an issue where properties are not fully copied by moving them after component node can be copied

Commit:
Merge branch 'develop' of github.com:headwirecom/peregrine-cms into develop

Commit:
issues/105: Fix for "NoSuchNodeTypeException: per:Replication" (#225)

Commit:
Excluded S3 tests from IT tests

Commit:
Merge branch 'develop' of github.com:headwirecom/peregrine-cms into develop

Commit:
#165 - Enhanced the application of given properties to apply child nodes as well

Commit:
#230 - Images with spaces / special character is failing the preview

because the Rendition Servlet checks if the resource matches the request name but the request name was not decoded.

Commit:
#231 - Fix for Images with special character during Asset Upload

This fix will convert any asset names from uploads if to UTF-8 when not that character encodiing

Commit:
#192 - By default new Assets will be replaced if they already exists

reusr1 committed:
#195: fix for datetime picker popping up modal when in preview mode

bkheadwire committed:
Removed the miliseconds from the ECMA date format since JS appears unable to digest them.

bkheadwire committed:
Merge pull request #232 from headwirecom/issues/195

#195: fix for datetime picker popping up modal when in preview mode

reusr1 committed:
#233: added mixin for created date to asset

bkheadwire committed:
Merge pull request #235 from headwirecom/issues/195

#233: added mixin for created date to asset

Commit:
Backport of the fixes for the Container / Multi Components Copy

Commit:
Merge branch 'develop' of github.com:headwirecom/peregrine-cms into develop

Commit:
Merge branch 'metaproc-develop' into METDEV-1362

* metaproc-develop: (96 commits)
  METDEV-1665 added translation for template component overlay and pages and fixed and that prevented the translation of materialicondropdown
  METDEV-1797: Fix bug in material-input
  METDEV-1797 bugfixing
  METDEV-1655: add translation for screen-dropdown
  METDEV-1564: Extend material-input
  METDEV-1656: remove wrong translation
  METDEV-1687: Remove unused favicon tags
  METDEV-1687: Remove favicons from peregrine-cms
  METDEV-1687: Externalize the favicon tags into its own html file
  METDEV-1687: Add usage of favicons for actual pages
  METDEV-1687: Fix location of favicons
  METDEV-1687: Add usage of favicons
  METDEV-1687: Add favicons
  METDEV-1655: deposit a german translation for save and cancel button in template editorial mask
  METDEV-1655: deposit a german translation for a headline in template editorial mask
  METDEV-1655: deposit a german translation for all screen buttons in template editor
  METDEV-1658: Move shim.min.js to pagerenderer module
  METDEV-1654 fix [visible] attribute not working anymore
  METDEV-1564: Fix class for controls-field
  METDEV-1655: deposit a german translation for the overlay menu of a component
  ...

Commit:
Merge branch 'develop' of https://github.com/headwirecom/peregrine-cms into METDEV-1362

# Conflicts:
#	admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/componentexplorer/template.vue
#	admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/pageview/template.vue
#	admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/toolingpage.html

reusr1 committed:
#157: made input type=range 5px smaller to avoid triggering the scrollbar

bkheadwire committed:
Fixed missing assets folder after site creation. (#238)

bkheadwire committed:
Issues/149 (#239)

* Fixed missing assets folder after site creation.
* Changed pathbrowser to make navigating into a folder the same as selecting it.

reusr1 committed:
fix package lock file

flx-sta committed:
headline field (feature) (#108)

Commit:
Merge branch 'develop' of https://github.com/headwirecom/peregrine-cms into METDEV-1362

# Conflicts:
#	admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/componentexplorer/template.vue
#	admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/pageview/template.vue
#	admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/toolingpage.html

DimitriPlotnikov commented:
@Felix-Puetz Is the branch still relevant or can we delete/close it?

flx-sta commented:
@DimitriPlotnikov relevant. @FabianHaupt testet darauf aktuell

DimitriPlotnikov commented:
OK

flx-sta commented:
Finally 👍👍
